shopBuddy/shopBuddy/Base.lproj/Main.storyboard

```diff
@@ -92,14 +101,114 @@
             </objects>
             <point key="canvasLocation" x="2860.8695652173915" y="97.767857142857139"/>
         </scene>
+        <!--Add Item View Controller-->
+        <scene sceneID="gsS-PU-lfF">
+            <objects>
+                <viewController id="DU9-0J-PMc" customClass="AddItemViewController" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="ayh-ZQ-sXa">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <button opaque="NO" contentMode="scaleToFill" fixedFrame="YES" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="CFI-dU-HwV">
+                                <rect key="frame" x="140" y="159" width="134" height="41"/>
+                                <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                <state key="normal" title="Button"/>
+                                <buttonConfiguration key="configuration" style="plain" title="Get Sample Item"/>
+                                <connections>
+                                    <action selector="didTapGetItem:" destination="DU9-0J-PMc" eventType="touchUpInside" id="NNU-VN-M8v"/>
+                                </connections>
+                            </button>
+                        </subviews>
+                        <viewLayoutGuide key="safeArea" id="l9D-6K-XzS"/>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor"/>
+                    </view>
+                    <connections>
+                        <segue destination="81r-ja-Gg9" kind="show" identifier="showDetailSegue" id="op9-hX-QNT"/>
+                    </connections>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="xEe-r7-aCG" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="3521.739130434783" y="97.767857142857139"/>
+        </scene>
+        <!--Item Detail View Controller-->
+        <scene sceneID="xaF-mn-pAD">
+            <objects>
+                <viewController id="81r-ja-Gg9" customClass="ItemDetailViewController" sceneMemberID="viewController">
```

Comment:
Is this view controller intentionally rendered as a modal?

Reply:
yes! I wanted to make it a modal bottom sheet later, so this is just temporary.